README.md

```diff
@@ -156,36 +156,37 @@ It is recommended that developers use the following scripts defined in `package.
-| `migrations:rollback`                   | Rolls all migrations back using their exported `down` functions.                                                                                              |
-| `migrations:up`                         | Runs one migration `up`.                                                                                                                                      |
-| `migrations:down`                       | Runs one migration `down`.                                                                                                                                    |
-| `scripts:run`                           | Runs a script. Usage: `npm run scripts:run -- <SCRIPT_NAME> [...args]`. Ensure the `--` is included to allow script arguments to be properly parsed.          |
-| `typedoc:build`                         | Builds all TypeDoc API documentation.                                                                                                                         |
-| `typedoc:start`                         | Serves TypeDoc documentation on a local server.                                                                                                               |
-| `docs:readme-toc`                       | Generate and insert a table of contents for `README.md`.                                                                                                      |
-| `docs:licenses`                         | Generate the list of licenses from this project's NPM dependencies in `docs/OPEN_SOURCE_LICENSES.txt`.                                                        |
-| `docs:db -- <POSTGRESQL_URL>`           | Generate database schema documentation in `docs/DATABASE.md` from the database specified by the connection url.                                               |
+| Script Name                             | Description                                                                                                                                                                                                        |
+| --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `cypress:run`                          | Runs the Cypress tests. (You must first manually start the app for the tests to have something to run against.) NOTE: The test set up and clean up will wipe and recreate the local database.                      |
+| `cypress:open`                          | Opens the interactive Cypress test runner. (You must first manually start the app for the tests to have something to run against.) NOTE: The test set up and clean up will wipe and recreate the local database.   |
```

Comment:
These are the only scripts I changed; the other changes are just making the table wider